metrics/cli/lib/deploy/deployer.dart

```diff
@@ -72,6 +73,16 @@ class Deployer {
       );
       await _installNpmDependencies();
       await _flutterService.build(DeployConstants.webPath);
+      _firebaseService.upgradeBillingPlan(projectId);
+      _firebaseService.enableAnalytics(projectId);
+      _firebaseService.initializeFirestoreData(projectId);
+
+      final googleClientId = _firebaseService.configureAuthProviders(projectId);
+      final configEnvironment = <String, dynamic>{
+        ConfigConstants.googleSignInClientId: googleClientId,
+      };
```

Comment:
Would it be great to have a `MetricsConfig` class with the `toMap` method to store the configuration?